privacyidea/lib/event.py

```diff
@@ -187,7 +228,8 @@ def set_event(name, event, handlermodule, action, conditions=None,
         id = int(id)
     event = EventHandler(name, event, handlermodule, action,
```

Comment:
What do you think about validating whether ``position`` is actually "pre" or "post" here?

Reply:
Will do later one day

Reply:
:+1: